package.js

```diff
@@ -1,20 +1,20 @@
 Package.describe({
   name: "jazeee:spiderable-longer-timeout",
   summary: "Extended spiderable package: SSL, caching, longer timeout, no stdin issues, publish flag",
-  version: "1.2.8",
+  version: "1.2.9",
   git: "https://github.com/jazeee/jazeee-meteor-spiderable"
 });
 
 Package.onUse(function (api) {
   api.versionsFrom("METEOR@0.9.0");
-  api.use(['webapp', 'mongo@1.1.0', 'ostrio:meteor-root@1.0.0', 'sha', 'cmather:handlebars-server@0.2.0'], 'server');
```

Comment:
@jazeee 
[`mongo@1.1.1`](https://atmospherejs.com/meteor/mongo)
[`ostrio:meteor-root@1.0.0`](https://atmospherejs.com/ostrio/meteor-root)
[`cmather:handlebars-server@2.0.0`](https://atmospherejs.com/cmather/handlebars-server)

As `handlebars-server` has major update and no changes description id created [this ticket](https://github.com/cmather/meteor-handlebars-server/issues/31), but I think it's cool and we are good to go.